doc/user-guides/secure-protect-connect.md

```diff
@@ -0,0 +1,372 @@
+- [Define `DNSPolicy` to bring traffic to the gateway](#define-dnspolicy)
+- [Override the Gateway's Deny ALL `AuthPolicy`](#override-the-gateways-deny-all-authpolicy)
+- [Override the Gateway `RateLimits`](#override-the-gateways-ratelimits) 
+
+
+To help with this walk through, you should set a `KUADRANT_ZONE_ROOT_DOMAIN` environmental variable to a domain you want to use. If it you want to try `DNSPolicy` this should also be a domain you have access to the DNS for in `route53 or GCP`. Example:
+```export KUADRANT_ZONE_ROOT_DOMAIN=my.domain.iown```
+
+### Deploy the example app we will serve via our gateway
+
+`kubectl apply -f https://raw.githubusercontent.com/Kuadrant/kuadrant-operator/main/examples/toystore/toystore.yaml`
+
+### Define a new Istio managed gateway
+
+```
+kubectl --context kind-kuadrant-local apply -f - <<EOF
+apiVersion: gateway.networking.k8s.io/v1
+kind: Gateway
+metadata:
+  name: api-gateway
+  namespace: kuadrant-system
+spec:
+  gatewayClassName: istio
+  listeners:
+  - allowedRoutes:
+      namespaces:
+        from: All
+    name: api
+    hostname: "*.$KUADRANT_ZONE_ROOT_DOMAIN"
+    port: 443
+    protocol: HTTPS
+    tls:
+      mode: Terminate
+      certificateRefs:
+        - name: apps-hcpapps-tls
+          kind: Secret
+EOF
+```
+
+
+If you take a look at the gateway status you will see a tls status error something like: 
+` message: invalid certificate reference /Secret/apps-hcpapps-tls. secret kuadrant-system/apps-hcpapps-tls not found`. This is because there is currently not TLS secret in place. Lets fix that with a `TLSPolicy`
+
+### Define TLSPolicy
+
+Note: For convenience in the setup, we have setup a self signed CA as a cluster issuer on the kind cluster.
+
+```
+kubectl --context kind-kuadrant-local apply -f - <<EOF
+apiVersion: kuadrant.io/v1alpha1
+kind: TLSPolicy
+metadata:
+  name: api-gateway-tls
+  namespace: kuadrant-system
+spec:
+  targetRef:
+    name: api-gateway
+    group: gateway.networking.k8s.io
+    kind: Gateway
+  issuerRef:
+    group: cert-manager.io
+    kind: ClusterIssuer
+    name: glbc-ca
+EOF
+
+kubectl wait tlspolicy api-gateway-tls -n kuadrant-system --for=condition=ready
+
+```
+
+Now if you look at the status of the gateway you will see the error is gone and the status of the policy you will see the listener is now secured with a tls certificate and the gateway is marked as affected by the tls policy. Our communication with our gateway is now secured via TLS. Note any new listeners will also be handled by the `TLSPolicy`
+
+
+Lets define a HTTPRoute and test our policy. We will re-use this with some of the other policies.
+
+
+
+```
+
+export INGRESS_PORT=$(kubectl get gtw api-gateway -o jsonpath='{.spec.listeners[?(@.name=="api")].port}' -n kuadrant-system)
+
+export INGRESS_HOST=$(kubectl get gtw api-gateway -o jsonpath='{.status.addresses[0].value}' -n kuadrant-system)
+
+export GATEWAY_URL=$INGRESS_HOST:$INGRESS_PORT
```

Comment:
I don't see this used anywhere. Same for `INGRESS_PORT`.

Reply:
yeah, think it can be removed, `GATEWAY_URL` was used throughout in an earlier iteration